src/simulator/interpret.cpp

```diff
@@ -1234,30 +1234,48 @@ namespace CoreIR {
     if ((clkVal->lastValue() == 0) &&
         (clkVal->value() == 1)) {
 
-      if (inSels.size() == 2) {
 
         setRegister(inst->toString(), bv1); //s1->getBits());
         ASSERT(same_representation(getRegister(inst->toString()),bv1),inst->toString() + " != " + toString(bv1)); //s1->getBits());
 
-      } else {
-        assert(inSels.size() == 3);
+      // TODO: Unused enable logic, remove?
```

Comment:
Since `ASSERT(inSels.size() == 2, ...)` was above, this code path wasn't being used, so I commented it out for now (I think enables are handled as explicit muxes?).  I can also remove it.

Reply:
I went ahead and removed this.